tests/std/tests/P0122R7_span/test.cpp

```diff
@@ -761,13 +787,10 @@ constexpr bool test() {
 
         FunctionTakingSpan<int>(orig_dyn);
         FunctionTakingSpan<int>(orig_three);
-        FunctionTakingSpan<int, 3>(orig_three);
         FunctionTakingSpan<const int>(orig_dyn);
         FunctionTakingSpan<const int>(orig_three);
         FunctionTakingSpan<const int>(orig_const_dyn);
         FunctionTakingSpan<const int>(orig_const_three);
-        FunctionTakingSpan<const int, 3>(orig_three);

```

Comment:
This should still compile, and it is important to test. Here, we have a converting construction from `span<int, 3>` to `span<const int, 3>`. That's a different element type, but the same extent. As the source extent isn't dynamic, this converting construction should be implicit.